lms/views/lti/basic_launch.py

```diff
@@ -294,8 +282,35 @@ def _show_document(
             extra=assignment_extra,
         )
 
+        # An assignment has been configured in the LMS as "gradable" if it has
+        # the `lis_outcome_service_url` param
+        gradable = bool(self.context.lti_params.get("lis_outcome_service_url"))
```

Comment:
It looks goofy leaving the `gradable` value out here, but we want to store this against the assignment.

The gradability of an assignment does not depend on whether we show you the interface or not.